src/TodoUpdateForm.tsx

```diff
@@ -23,13 +31,9 @@ const TodoUpdateForm = ({ title, onSubmit }: Props) => {
           value={inputText}
           onChange={(e) => inputTextChange(e)}
         />
-        <button
-          type="button"
-          className="btn-type-01 green"
-          onSubmit={() => onSubmit}
-        >
+        <Button variants="update-save" onClick={() => onAction()}>
```

Comment:
minor) onAction은 이미 함수 이름이 아닌가요?

아래 처럼 바꿔볼 수 있을 것 같아요~

```suggestion
        <Button variants="update-save" onClick={onAction}>
```